ofctrl/ofSwitch.go

```diff
@@ -357,6 +357,8 @@ func (self *OFSwitch) handleMessages(dpid net.HardwareAddr, msg util.Message) {
 				log.Errorf("Received Vendor error: %s on ONFT_BUNDLE_ADD_MESSAGE message", errMsg)
 			}
 		default:
+			result.msgType = UnknownMessage
+			self.publishMessage(0, result)
```

Comment:
@weiqiangt where will the message be received since it doesn't have a valid bundleID?

Reply:
Sorry, I used to wrongly assumed the xID is different from the bundle ID.